docs/cspell.json

```diff
@@ -0,0 +1,699 @@
+"etcdctl",
+"plugindata",
+"Println",
+"Sprintf",
+"watcherjob",
+"pseudoversion",
+"allowedlogins",
+"attrname",
+"datadoghq",
+"flexi",
+"idps",
+"myns",
+"prioritise",
+"SAMLIDP",
+"samltest",
+"samltestid",
+"scrollback",
+"vkxz",
+"XNCK",
+"cond",
+"FGHIJ",
+"replaceall",
+"anotheruser",
+"auths",
+"CHANGEME",
+"cimg",
+"circleci",
+"CLOUDSDK",
+"clusterolebinding",
+"deregisters",
+"devicetrust",
+"gsutil",
+"IAMR",
+"initcontainers",
+"mermaidjs",
+"netcat",
+"nosql",
+"obtainlicense",
+"octocat",
+"pgaadauth",
+"rdsproxy",
+"rollouts",
+"rtrzn",
+"uninstallation",
+"workgroups",
+"yourdatabase",
+"ioreg",
+"ndots",
+"GHES",
+"backoff",
+"NOKEY",
+"awsapp",
+"acpi",
+"clis",
+"httpout",
+"localca",
+"myproject",
+"MYTOKEN",
+"navin",
+"nvme",
+"priya",
+"splunkd",
+"splunkforwarder",
+"tcpout",
+"timechart",
+"westeurope",
+"splunkd",
+"Traefik",
+"xeyes",
+"dockerized",
+"cloudkms",
+"mycluster",
+"myrolebinding",
+"Rdik",
+"serviceaccountname",
+"storageenabled",
+"xlink",
+"actionstest",
+"auditctl",
+"auditd",
+"ausearch",
+"authserver",
+"cqlsh",
+"datareader",
+"keyfile",
+"Keyspaces",
+"loginuid",
+"managedclusters",
+"mypartial",
+"roadmap",
+"rolearn",
+"runcommand",
+"selfsubjectrulesreviews",
+"tokenconfig",
+"truststore",
+"auditctl",
+"auditd",
+"authserver",
+"AADUSER",
+"ABCDEFGHIJKL",
+"abcdefghijklm",
+"aclfile",
+"adcli",
+"Addrs",
+"ADFS",
+"adutil",
+"Afax",
+"agentless",
+"agentpool",
+"AICPA",
+"AICPA’s",
+"AKIA",
+"allcommands",
+"allkeys",
+"allowdeny",
+"AMAZ",
+"apikey",
+"apiserver",
+"appdomain",
+"APPSECRET",
+"appuser",
+"AQAB",
+"Aqxs",
+"Archlinux",
+"armv",
+"atburke",
+"authc",
+"AUTHINFO",
+"authkey",
+"authservers",
+"Authy",
+"authz",
+"automount",
+"autoscale",
+"awscli",
+"awsconsole",
+"awsdatabases",
+"awsuser",
+"azuread",
+"azuredatabases",
+"backporting",
+"backports",
+"BDRVJUSUZ",
+"behaviour",
+"bfhunter",
+"Binm",
+"bjoerger",
+"bluemix",
+"boto",
+"BSFD",
+"BUCKETNAME",
+"buildbox",
+"caddef",
+"categorisation",
+"cavium",
+"CCDC",
+"centralus",
+"certificatekey",
+"certutil",
+"cfsdf",
+"Cgajq",
+"cgroupv",
+"chacha",
+"cicd",
+"ciphersuites",
+"clientid",
+"cloudbuild",
+"cloudsql",
+"clusers",
+"clusteradmin",
+"clustername",
+"clusterrole",
+"clusterrolebinding",
+"cockroachdb",
+"codingllama",
+"configmap",
+"connectorname",
+"createkey",
+"creds",
+"crond",
+"CTAP",
+"customizability",
+"journalctl",
+"datacenter",
+"dbaccessdemo",
+"dbadir",
+"dbadmin",
+"dbeaver",
+"dbgroup",
+"dbname",
+"DBSIZE",
+"dbuser",
+"DEBU",
+"Decisiv",
+"Deeplink",
+"devel",
+"develnode",
+"devs",
+"Dfnnpu",
+"Dfumu",
+"DHDR",
+"disablesse",
+"distros",
+"Divio's",
+"dockerhost",
+"doctl",
+"dodemo",
+"domainwidedelegation",
+"DQMB",
+"dronegen",
+"dsacls",
+"DSID",
+"dspublish",
+"dualstack",
+"dylib",
+"dynamicappregexample",
+"dynamicregexample",
+"eastus",
+"ECMWF",
+"editbasicsaml",
+"efgh",
+"efghijk",
+"ekontsevoy",
+"eksctl",
+"Elastcsearch",
+"Elasticvue",
+"elbv",
+"elbz",
+"elif",
+"emtpy",
```

Comment:
I think this is probably a mispelling that should be updated in the docs?

Reply:
This is from the terraform code blocks, docs/pages/deploy-a-cluster/deployments/aws-terraform.mdx

```code
if [[ "${INSTANCE_TYPE}" == "auth" ]]; then
    export SERVER_IP=$(aws ec2 describe-instances --filters "Name=tag:TeleportCluster,Values=${TF_VAR_cluster_name}" "Name=tag:TeleportRole,Values=auth" --query "Reservations[${INSTANCE_ID}].Instances[*].PrivateIpAddress" --output text)
    echo "Auth ${INSTANCE_ID} IP: ${SERVER_IP}"
elif [[ "${INSTANCE_TYPE}" == "proxy" ]]; then
    export SERVER_IP=$(aws ec2 describe-instances --filters "Name=tag:TeleportCluster,Values=${TF_VAR_cluster_name}" "Name=tag:TeleportRole,Values=proxy" --query "Reservations[${INSTANCE_ID}].Instances[*].PrivateIpAddress" --output text)
    echo "Proxy ${INSTANCE_ID} IP: ${SERVER_IP}"
elif [[ "${INSTANCE_TYPE}" == "node" ]]; then
    export SERVER_IP=$(aws ec2 describe-instances --filters "Name=tag:TeleportCluster,Values=${TF_VAR_cluster_name}" "Name=tag:TeleportRole,Values=node" --query "Reservations[*].Instances[*].PrivateIpAddress" --output text)
    echo "Node IP: ${SERVER_IP}"
fi
```

Reply:
Yes, ./pages/management/guides/joining-nodes-aws-iam.mdx:create and attach an emtpy role with no attached policies.